api/src/db/migrations/v157_add_tax_clearance_data.ts

```diff
@@ -0,0 +1,971 @@
+import { v156Business, v156UserData } from "@db/migrations/v156_remove_in_progress_task_status";
+import { randomInt } from "@shared/intHelpers";
+
+export const migrate_v156_to_v157 = (v156Data: v156UserData): v157UserData => {
+  return {
+    ...v156Data,
+    businesses: Object.fromEntries(
+      Object.values(v156Data.businesses)
+        .map((business: v156Business) => migrate_v156Business_to_v157Business(business, v156Data))
+        .map((currBusiness: v157Business) => [currBusiness.id, currBusiness])
+    ),
+    version: 157,
+  } as v157UserData;
+};
+
+export const migrate_v156Business_to_v157Business = (
+  business: v156Business,
+  userData: v156UserData
+): v157Business => {
+  return {
+    ...business,
+    version: 157, // Do not remove
+    versionWhenCreated: 157, // Do not remove
+    userId: userData.user.id, // Do not remove
```

Comment:
Are these your comments or artifacts from the migration generator?

Reply:
I added the comments, I'm not sure if the values need to be added for every change.We only need version, right?

Reply:
removed comments